### DISCUSSION TITLE
Home Assistant nodes are not available

### DISCUSSION THREAD

**Discussion by mareklac** [zachowj/node-red-contrib-home-assistant-websocket#566] [Q&A]
I have installed Node-RED addons in homeassistant. Addons is working correctly ..no issues in log. However nodes from HA are not available . I checked that websocket is also installed but it is disabled and can not enable it at all .. I think this is a reason that HA nodes are not available. How make these nodes available and working ?

**Comment by zachowj** [comment]
Not following exactly what you have installed and what's not working.

Are you running NR as a stand-alone or as a [Home Assistant Add-on](https://github.com/hassio-addons/addon-node-red)?

**Reply by mareklac** [reply]
I run it as Home Assistant Add-on. 
I tried to make first automatization using NR but on left side panel there are missing set of HA nodes e.g. ha-entity, trigger-state ... all HA related. 
Other nodes e.g inject, debug , function or switch are available

**Reply by zachowj** [reply]
What do your NR logs show? Try uninstalling and reinstalling the addon-on. This also sounds more of an add-on issue than this package so you might get better help at the add-on repo linked above.

**Comment by mareklac** [comment]
In log nothing special :

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing...
----------------------------------------------------------- Add-on:
Node-RED Flow-based programming for the Internet of
Things-----------------------------------------------------------
Add-on version: 10.4.0 You are running the latest version of this
add-on. System: Home Assistant OS 7.2  (armv7 / raspberrypi3) Home
Assistant Core: 2021.12.10 Home Assistant Supervisor:
2022.01.1-----------------------------------------------------------
Please, share the above information when looking for help or support
in, e.g., GitHub, forums or the Discord
chat.-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing...
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] customizations.sh: executing...
[cont-init.d] customizations.sh: exited 0.
[cont-init.d] nginx.sh: executing...
[cont-init.d] nginx.sh: exited 0.
[cont-init.d] node-red.sh: executing...
patching file nodes/ui_base.html
Hunk #1 succeeded at 1162 (offset 631 lines).
up to date, audited 1 package in 3s
found 0 vulnerabilities
[cont-init.d] node-red.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[18:01:29] INFO: Starting Node-RED...
> start
> node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"
1 Feb 18:01:41 - [info]
Welcome to Node-RED
===================
1 Feb 18:01:41 - [info] Node-RED version: v2.2.0
1 Feb 18:01:41 - [info] Node.js  version: v16.13.2
1 Feb 18:01:41 - [info] Linux 5.10.63-v7 arm LE
1 Feb 18:01:43 - [info] Loading palette nodes


I noticed that the problem occurs in FireFox .. in Microsoft Edge all node
types are available

pon., 31 sty 2022 o 17:31 Jason ***@***.***> napisał(a):
